modules/importing-repositories/procedure-enabling-the-bulk-import-from-github-feature.adoc

```diff
@@ -0,0 +1,46 @@
+[id="enabling-ang-giving-access-to-the-bulk-import-feature"]
+= Enabling and giving access to the Bulk Import feature
+
+.Prerequisites
+. You have link:{authentication-book-url}#enabling-authentication-with-github[configured GitHub authentication and integration].
+
+.Procedure
+
+. The Bulk Import plugins are installed but disabled by default.
+To enable the `./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import-backend-dynamic` and `./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import` plugins,
+edit your `dynamic-plugins.yaml` with following content.
++
+.`dynamic-plugins.yaml` fragment
+[source,yaml]
+----
+plugins:
+  - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import-backend-dynamic
+    disabled: false
+  - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import
+    disabled: false
+----
++
+See link:{installing-and-viewing-dynamic-plugins-url}[{installing-and-viewing-dynamic-plugins-title}].
+
+. Only {product-short} administrators or users with the `bulk.import` permission can use the Bulk Import feature.
+Configure the required `bulk.import` RBAC permission for the users that are not administrators.
++
+.`rbac-policy.csv` fragment
+[source,csv,subs="+quotes"]
+----
+p, role:default/bulk-import, bulk.import, use, allow
+p, role:default/bulk-import, catalog-entity.read, read, allow
+p, role:default/bulk-import, catalog.entity.create, create, allow
+g, user:default/__<your_user>__, role:default/bulk-import
+----
++
+See link:{authorization-book-url}#ref-rbac-permission-policies_title-authorization[Permission policies in Red Hat Developer Hub].
+
+.Verification
+* The sidebar displays a *Bulk Import* line.
+* The *Bulk Import* page shows a list of *Added Repositories*.
+
+.Troubleshooting
+* If clicking on the *Bulk Import* option displays the following error message: 
+_To view the added repositories, contact your administrator to give you the `bulk.import` permission_, which means the user does not have the necessary permission to use the Bulk Import feature.
+See link:{authorization-book-url}#ref-rbac-permission-policies_title-authorization[Permission policies in Red Hat Developer Hub] for more information on how an administrator can add that permission to your user.
```

Comment:
Removing. It repeats the last procedure step, and the error message is self-sufficient.

```suggestion
```